src/Ledger/Script.lagda

```diff
@@ -71,36 +73,44 @@ module _ ⦃ _ : DecEq Slot ⦄ (_≤_ : Slot → Slot → Set) (_≤ᵇ_ : Slot
                    a l r : Slot
 \end{code}
 \begin{code}
-  data evalTimelock (khs : ℙ KeyHash) (I : Maybe Slot × Maybe Slot) : Timelock → Set where
-    evalAll : All (evalTimelock khs I) ss → evalTimelock khs I (RequireAllOf ss)
-    evalAny : Any (evalTimelock khs I) ss → evalTimelock khs I (RequireAnyOf ss)
-    evalMOf : ss' S.⊆ ss → All (evalTimelock khs I) ss' → evalTimelock khs I (RequireMOf (length ss') ss)
-    evalSig : x ∈ khs → evalTimelock khs I (RequireSig x)
-    evalTSt : proj₁ I ≡ just l → a ≤ l → evalTimelock khs I (RequireTimeStart a)
-    evalTEx : proj₂ I ≡ just r → r ≤ a → evalTimelock khs I (RequireTimeStart a)
+  module _ (khs : ℙ KeyHash) (I : Maybe Slot × Maybe Slot) where
+    data evalTimelock : Timelock → Set where
+      evalAll  : All evalTimelock ss
+               → evalTimelock (RequireAllOf ss)
+      evalAny  : Any evalTimelock ss
+               → evalTimelock (RequireAnyOf ss)
+      evalMOf  : ss' S.⊆ ss → All evalTimelock ss'
+               → evalTimelock (RequireMOf (length ss') ss)
+      evalSig  : x ∈ khs
+               → evalTimelock (RequireSig x)
+      evalTSt  : I .proj₁ ≡ just l → a ≤ l
+               → evalTimelock (RequireTimeStart a)
+      evalTEx  : I .proj₂ ≡ just r → r ≤ a
+               → evalTimelock (RequireTimeStart a)
```

Comment:
@WhatisRT How about this use of `module`? It seems that, although a more "advanced" feature than plain datatypes, it will be more readable to a non-Agda person.

Reply:
Yeah, I think this is fine. Especially since we can do some trickery with LaTeX, where we hide the line containing the `module` keyword, and replace it by prose such as 'in the following, `khs` and `I` are ...'.